main/helpcontent2/source/text/scalc/01/04060104.xhp

```diff
@@ -637,9 +637,9 @@
 <tablecell>
 <paragraph role="tablecontent" id="par_id3151328" xml-lang="en-US" l10n="U" oldref="179">Returns the file name and the sheet number of the referenced cell.</paragraph>
 <paragraph role="tablecontent" id="par_id3148896" xml-lang="en-US" l10n="U" oldref="180">
-<item type="input">=CELL("FILENAME";D2)</item> returns 'file:///X:/dr/own.sxc'#$Sheet1, if the formula in the current document X:\dr\own.sxc is located in Sheet1.</paragraph>
+<item type="input">=CELL("FILENAME";D2)</item> returns 'file:///X:/dr/own.ods'#$Sheet1, if the formula in the current document X:\dr\own.sxc is located in Sheet1.</paragraph>
```

Comment:
There is still a missing replacement here, near the end of the string.

Reply:
And basically this PR is the same as PR #25
I don't know why this was duplicated?

Reply:
Should be fixed now!